src/plugins/dashboard_embeddable_container/public/embeddable/viewport/dashboard_viewport.test.tsx

```diff
@@ -33,6 +33,10 @@ import {
   ContactCardEmbeddableFactory,
 } from '../../embeddable_plugin_test_samples';
 import { KibanaContextProvider } from '../../../../../plugins/kibana_react/public';
+import {
+  DashboardEmptyScreen,
+  DashboardEmptyScreenProps,
+} from '../../../../../legacy/core_plugins/kibana/public/dashboard/dashboard_empty_screen';
```

Comment:
We cannot import from `/src/legacy` from within a New Platform plugin `/src/plugins`? Can we move this `dashboard_empty_screen` to the new platform, say `/src/plugins/dashboard` plugin?